src/types/options/index.ts

```diff
@@ -152,20 +152,14 @@ export interface BlockOptions {
   options: Option[];
 }
 export interface MessageOptions {
-  options: {
-    text: string;
-    value: string;
-  }[];
+  options: Option[];
 }
 export interface DialogOptions {
-  options: {
-    label: string;
-    value: string;
-  }[];
+  options: Option[];
 }
 export interface OptionGroups<Options> {
```

Comment:
If the option group here is used for both legacy dialog and block kit based external data select, perhaps we need to have the two (meaning DialogOptionGroups and OptionGroup) instead.

Reply:
I added the `DialogOptionGroups<Options>` interface to accommodate these different types, with differences being in the `label` type. I opted to follow a similar pattern as `OptionGroups<Options>` (by including the generic), but if you'd think explicitly using the `DialogOptions` for `options` would be a better pattern (or something else), I'm happy to change this! For example:

```typescript
export interface DialogOptionGroups {
  option_groups: ({
    label: string;
    options: DialogOptions[];
  })[];
}
```